resources/views/service/create_registration.blade.php

```diff
@@ -42,18 +42,18 @@
                         </button>
                     </div>
                 </div>
-                <div class="collectors">
-                    <p>These people may collect vouchers:</p>
+                <div class="added">
+                    <p>You have added:</p>
```

Comment:
This is a P, wheras the "added" for kids is an H2, the sizing looks strange.

Reply:
good spot! Was going to fix the styling after we get the designer's stuff through, but this should deffo be a h2. Thanks!

Reply:
bodge in an H2 for now and we'll rewrite when they give us the pix.